contrib/packs/actions/auto_deploy_repo.yaml

```diff
@@ -0,0 +1,18 @@
+---
+  description: "Check if a given branch in a ST2 Pack's Git repository should be auto deployed"
+  enabled: true
+  entry_point: 'auto_deploy_repo.py'
+  name: "auto_deploy_repo"
```

Comment:
to correctly capture the nature of this action perhaps `check_auto_deploy_repo` would be a better name.